src/Gaufrette/Adapter/PhpseclibSftp.php

```diff
@@ -155,7 +155,7 @@ public function delete($key)
     /**
      * {@inheritdoc}
      */
-    public function createFile($key, Filesystem $filesystem)
+    public function createFile($key, FilesystemInterface $filesystem)
     {
         $file = new File($key, $filesystem);
```

Comment:
Be careful, the signature of `File` does not accept a `FilesystemInterface` as the second parameter but a `Filesystem`. Can you also update it please ?

Reply:
This is already done.